handsontable/src/plugins/manualColumnResize/manualColumnResize.js

```diff
@@ -396,6 +397,11 @@ export class ManualColumnResize extends BasePlugin {
       return;
     }
 
+    // A "mouseover" action is triggered right after executing "contextmenu" event. It should be ignored.
+    if (this.skipMouseOverAction === true) {
```

Comment:
WDYT to rename that "fuse" to `this.isTriggeredByContextMenu` or something? The name `skipMouseOverAction` does not say why the action is skipped. The name seems too generic, requiring looking for the code that uses that flag to get the point.

Reply:
Done within https://github.com/handsontable/handsontable/pull/10416/commits/b7ca869936281bac45ddc7dbb33da23193f5b022.

Reply:
Thanks! Can I ask one more time for a fix? There is a typo `this.isTriggeredbyRMB` should be `this.isTriggeredByRMB`.

Reply:
Done within https://github.com/handsontable/handsontable/pull/10416/commits/edf886b43287eb0b784dda37710adeb4dab8f084.